src/v3/src/components/PasswordWithConfirmation/PasswordWithConfirmation.tsx

```diff
@@ -37,95 +37,116 @@ const PasswordWithConfirmation: UISchemaElementComponent<{
       messages: newPasswordMessages = {},
     },
   } = input.options;
-  const newPasswordError = newPasswordMessages?.value?.[0]
-    && getMessage(newPasswordMessages.value[0]);
+  const error = newPasswordMessages?.value?.[0] && getMessage(newPasswordMessages.value[0]);
+
+  console.log('aaaa', error);
+
+  
 
   const value = useValue(input);
-  const [isTouched, setIsTouched] = useState<boolean>(false);
+  // const [isTouched, setIsTouched] = useState<boolean>(false);
   const [confirmPassword, setConfirmPassword] = useState<string>();
-  const [confirmPasswordError, setConfirmPasswordError] = useState<string | undefined>(
-    fieldRequiredErrorMessage,
-  );
+  const [confirmPasswordError, setConfirmPasswordError] = useState<string | undefined>();
   const { dataSchemaRef } = useWidgetContext();
 
-  const handleConfirmPasswordValidation = (password: string | undefined): void => {
-    if (!password) {
-      setConfirmPasswordError(fieldRequiredErrorMessage);
-      return;
+  const validate = useCallback((data: FormBag['data']) => {
+    const password = data[input.name];
+
+    console.log('validate', password, confirmPassword);
+    
+    if (confirmPassword === password) {
+      return undefined;
```

Comment:
When you land on the page and immediately click submit without entering text in either field, wouldn't this submit a request? We should avoid an unnecessary request when we know the user hasn't fulfilled form requirements right?